src/EventProducerService.cpp

```diff
@@ -62,17 +56,16 @@ void EventProducerService::process(UserInterface::RequestedAction requestedActio
   }
 }
 
-void EventProducerService::sendEvent(bool state, byte evValue)
+void EventProducerService::sendEvent(bool state, byte index)
```

Comment:
You changed the parameter name "evValue" to "index" here. I guess you should do the same for the other sendEvent functions.